OrbitProducerSideChannel/include/OrbitProducerSideChannel/ProducerSideChannel.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_PRODUCER_SIDE_CHANNEL_PRODUCER_SIDE_CHANNEL_H_
+#define ORBIT_PRODUCER_SIDE_CHANNEL_PRODUCER_SIDE_CHANNEL_H_
+
+#include "grpcpp/grpcpp.h"
+
+namespace orbit_producer_side_channel {
+
+// This is the default path of the Unix domain socket used for the communication
+// between producers of CaptureEvents and OrbitService.
+constexpr const char* kProducerSideUnixDomainSocketPath = "/tmp/orbit-producer-side-socket";
```

Comment:
Personal preference and very nit: Maybe we could construct a std::string_view here directory?

Reply:
Done, I don't have a preference, mostly I'm still a bit lost about `string_view`. I though it was mostly for receiving a string rather than to pass a string around?

Reply:
Yes, that's the most common use-case. But it also works great as a string-literal replacement. And there is also a performance consideration: Most string literals (const char*) needs to be converted into a std::string at some point.
That requires std::strlen to be called to determine the length. But the length could already be known since it's static
and a literal. string_view solves that problem. It has a constexpr constructor and determines the length at compile time,
avoiding the strlen-call at runtime.

Reply:
Ah excellent, thanks for the explanation!